src/qibo/models/dbi/double_bracket.py

```diff
@@ -57,39 +57,53 @@ def __init__(
     def __call__(
         self, step: float, mode: DoubleBracketGeneratorType = None, d: np.array = None
     ):
+        r"""We use convention that $H' = U^\dagger H U$ where $U=e^{-sW}$ with $W=[D,H]$ (or depending on `mode` an approximation, see `eval_dbr_unitary`). If $s>0$ then for $D = \Delta(H)$ the GWW DBR will give a $\sigma$-decrease, see https://arxiv.org/abs/2206.11772."""
+
         operator = self.eval_dbr_unitary(step, mode, d)
         operator_dagger = self.backend.cast(
             np.matrix(self.backend.to_numpy(operator)).getH()
         )
-        self.h.matrix = operator @ self.h.matrix @ operator_dagger
```

Comment:
I switch them because in the `eval_dbr_unitary` function the minus sign is now added